services/ResourceService/src/index.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Islandora\ResourceService;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+use Silex\Application;
+use Islandora\Chullo\Chullo;
+use Islandora\Chullo\TriplestoreClient;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Psr\Http\Message\ResponseInterface;
+use Silex\Provider\TwigServiceProvider;
+use Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory;
+
+
+date_default_timezone_set('UTC');
+
+$app = new \Silex\Application();
+
+$app['debug'] = true;
+
+$app->register(new \Silex\Provider\TwigServiceProvider(), array(
+  'twig.path' => __DIR__.'/../templates',
+));
+
+$app['twig'] = $app->share($app->extend('twig', function($twig, $app) {
+
+  return $twig;
+}));
+
+$app['fedora'] =  $app->share(function() {
+
+  return Chullo::create('http://localhost:8080/rest');
+});
+
+$app['triplestore'] = $app->share(function() {
+
+  return TriplestoreClient::create('http://localhost:9999/bigdata/sparql');
+});
+
+$app['data.resourcepath'] = NULL;
+
+$getPathFromTriple = function (Request $request,\Silex\Application $app) {
+  // In case the request was made by a browser, avoid 
+  // returning the whole Fedora4 API Rest interface page.
+  if (0 === strpos($request->headers->get('Accept'),'text/html')) {
+    $request->headers->set('Accept', 'text/turtle', TRUE);
+  }
+  $sparql_query = $app['twig']->render('getResourceByUUIDfromTS.sparql', array(
+    'uuid' => $request->attributes->get('uuid'),
+  ));
+   
+  $sparql_result = $app['triplestore']->query($sparql_query);
+  // We only assign one in case of multiple ones
+  // Will have to check for edge cases?
+  foreach ($sparql_result as $triple) {
+    $app['data.resourcepath'] = $triple->s->getUri();
+  }
+  // Abort the routes if we don't get a subject from the tripple.
+  if (NULL === $app['data.resourcepath']) {
```

Comment:
Pretty much.  With PUT, we'd jam the UUID into the RDF if it doesn't already exist.  With POST, we'd be using the UUID to find the parent resource we're going to add to.  We'd have to generate a new UUID and stick that in the RDF if it doesn't already exist (maybe use the one in the RDF if it does already exist?)

For binaries, is it possible to add the uuid to a resource's fcr:metadata?  If it is, I'd do that.  That wouldn't affect what we're doing now and would be in line with where we want to go (one node per resource).